lib/retrieval-stats.js

```diff
@@ -83,6 +88,10 @@ export const buildRetrievalStats = (measurements, telemetryPoint) => {
 
     const node = `${m.inet_group}::${m.participantAddress}`
     tasksPerNode.set(node, (tasksPerNode.get(node) ?? 0) + 1)
+
+    const taskId = getTaskId(m)
+    if (m.indexerResult) tasksQueryingIndexer.add(taskId)
```

Comment:
what is a "task querying the indexer"? Or is it a task "from" querying the indexer?

Reply:
Let me explain. 

I want to calculate the answer to the following question: How many deals are advertising HTTP retrieval to IPNI?

I started with a naive approach = the fraction of measurements with `indexerResult: OK` from all measurements.

That does not work now because >95% of the measurements have node indexer results (they were submitted by old Spark checkers that don't query IPNI).

Also, we have a variable number of measurements for each task, ranging from ~7 to ~400. This distorts the calculated value, too.

To get a more precise value, I want to calculate the numbers for tasks, not measurements.

1. total = For how many tasks in this round do we have indexer query results?
2. http = For how many tasks in this round was there at least one indexer query result containing an HTTP retrieval advertisement?

The answer to my original question is then `http/total`.

I am using `tasksQueryingIndexer` to count `total`, and `tasksAdvertisingHttp` to count `http`.

I see how the current names can be confusing. Could you suggest better ones?

Reply:
I think the confusing part is that a task isn't querying the indexer. What about

- `tasksWithIndexerResults`
- `tasksWithHTTPAdvertisement`

these are based on your descriptions

> total = For how many tasks in this round do we have indexer query results?
http = For how many tasks in this round was there at least one indexer query result containing an HTTP retrieval advertisement?